dev/ci/ci-vst.sh

```diff
@@ -8,6 +8,10 @@ VST_CI_DIR="${CI_BUILD_DIR}/VST"
 # opam install -j ${NJOBS} -y menhir
 git_checkout "${VST_CI_BRANCH}" "${VST_CI_GITURL}" "${VST_CI_DIR}"
 
-# Targets are: msl veric floyd progs , we remove progs to save time
-# Patch to avoid the upper version limit
-( cd "${VST_CI_DIR}" && make IGNORECOQVERSION=true .loadpath version.vo msl veric floyd )
+# HACK: from the upstream makefile:
+#
+# default_target: _CoqProject version.vo msl veric floyd progs
+#
+# We have to omit progs as otherwise we timeout on Travis; once we
+# move to Gitlab we will able to just use `make`
```

Comment:
Unfortunately, that would mean also removing Travis support, which is easier to run for our contributors.

Reply:
>Unfortunately, that would mean also removing Travis support, which is easier to run for our contributors.

I think we will end up with a ci-vst-lite.sh and a ci-vst.sh for travis/gitlab.

Reply:
> Unfortunately, that would mean also removing Travis support, which is easier to run for our contributors.

Realistically Travis support will be phased out at some point, Gitlab is pretty to run by the way; they can just clone the main Coq repos there and hit "mirror".